app/services/list_collection.rb

```diff
@@ -1,9 +1,10 @@
 class ListCollection
+  include ActiveModel::Model
 
-  def initialize page: 1, page_size: 25
```

Comment:
I was trying to find a "clean" way to pass more data into it.
If it where rails 5 I'd use `ActiveModel::AttributeAssignment` which it's just what I need.